underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -159,6 +165,26 @@ public SwiftUnderFileSystem(AlluxioURI uri) {
       container.create();
     }
     mContainerPrefix = Constants.HEADER_SWIFT + mContainerName + PATH_SEPARATOR;
+
+    // Assume the Swift user name has 1-1 mapping to Alluxio username.
+    mAccountOwner = Configuration.get(PropertyKey.SWIFT_USER_KEY);
+    short mode = (short) 0;
+    // If there is any container ACL for the Swift user, translates it to Alluxio permission.
+    if (container.getContainerReadPermission().contains(mAccountOwner)
```

Comment:
Per our discussion, I think we should go with a more user-friendly (less-secure) way that once there's any `*` showing up in the ACL, we should relax the permission restriction and grant the account owner has the specific permission to the container. We can state this limitation explicitly in our docs. If any user need more specific regexp matching with their ACL policy, it should be easy for them to modify this.